### PR TITLE
Completed css-three-column-layout exercise. - Damien Martinez

### PR DESCRIPTION
**Pull request by damien-martinez**
Screen Shot 2022-01-27 at 7 46 08 PM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    u+2000-206f,
+    u+2074,
+    u+20ac,
+    u+2122,
+    u+2191,
+    u+2193,
+    u+2212,
+    u+2215,
+    u+feff,
+    u+fffd;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:26]: I didn't teach you any of these, they are unnecessary for you to use 😂

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css (hunk 1 above)
Comment (damien-martinez) [reply css-three-column-layout/styles.css]: > I didn't teach you any of these, they are unnecessary for you to use 😂

LOL Sorry! I thought we had to copy everything from the fonts.css to be able to use the font. I just took it off and re-submitted.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
+* {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  src:
+    url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2")
+    format("woff2");
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:8]: This can also be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-three-column-layout/styles.css (hunk 2 above)
Comment (damien-martinez) [reply css-three-column-layout/styles.css]: Thanks Cody. I just removed it at re-pushed. I will make to only keep the font family and when needed, font-weight.

Thanks again!

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**